nginx-auto-config.go

```diff
@@ -225,3 +229,15 @@ func takeInput() int {
 	}
 	return input
 }
+
+func verifyDirInput() string {
+	dirName := getInput(false, false)
+
+	if !dirExists(dirName) {
+		_, _ = red.Printf("Directory '%v' is non existent, please try again.\n", dirName)
+		_, _ = cyan.Print("Root path: ")
+		return verifyDirInput()
```

Comment:
It'd be better to do:
```
if dirExists(dirName) {
    return dirName
}
_, _ = red.Printf("Directory '%v' is non existent, please try again.\n", dirName)
_, _ = cyan.Print("Root path: ")
return verifyDirInput()
```

Reply:
Underlying functionality is exactly the same and hence would be unnecessary.

Reply:
We should avoid using else as much as possible, it's not very clean

Reply:
You may notice this is a general theme followed in the code

Reply:
Done.